os_net_config/impl_nmstate.py

```diff
@@ -170,6 +186,13 @@ def is_dict_subset(superset, subset):
 
 
 def _add_sub_tree(data, subtree):
+    """Create a nested dict with the keys mentioned in subtree
+
+    :param data: Starting point for the creation of nested dict
+    :param subtree: list of keys used in the creation of nested
+        dict.
+    :returns: The tip of the nested dict created
+    """
     if data is None:
```

Comment:
Should we mention the exceptions raised like "raises: ConfigErr" ?

Reply:
Done